editions/tw5.com/tiddlers/empty-tag-node-template.tid

```diff
@@ -1,11 +1,11 @@
 created: 20240710161501472
-list-after: $:/core/ui/ViewTemplate/body
-modified: 20240713020832439
+list-after: 
```

Comment:
Should `list-after` really be removed? Did you drag&drop sort the ViewTemplate for testing?

Reply:
I meant to leave the list-after field present, but with no value. That way, it should sort even below any other custom template (including ones I can't name) ;)

Reply:
OK. I just wanted to make sure it was intentional